### PR TITLE
Bump for DiffRules 1.0 compat

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
Should this just be to 1.0?

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "ForwardDiff"
 uuid = "f6369f11-7733-5829-9624-2563aa707210"
-version = "0.10.7"
+version = "0.11.0"
```
Comment (fredrikekre) [inline Project.toml]:
```suggestion
version = "0.10.8"
```
?

Comment (ChrisRackauckas) [reply Project.toml]: but it had a compatibility change

Comment (fredrikekre) [reply Project.toml]: Yes? How is that breaking?

Comment (ChrisRackauckas) [reply Project.toml]: Doesn't General require a 0.x.0 bump with compatibility changes?

Comment (fredrikekre) [reply Project.toml]: No.

Comment (ChrisRackauckas) [reply Project.toml]: Alrighty then.

**Pull request merged**